new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -0,0 +1,706 @@
+                        .image(WLS_BASE_IMAGE_NAME + ":" + WLS_BASE_IMAGE_TAG)
+                        .addCommandItem("/bin/sh")
+                        .addArgsItem("-c")
+                        .addArgsItem("chown -R 1000:1000 /shared")
+                        .volumeMounts(Arrays.asList(
+                            new V1VolumeMount()
+                                .name(pvName)
+                                .mountPath("/shared")))
+                        .securityContext(new V1SecurityContext()
+                            .runAsGroup(0L)
+                            .runAsUser(0L))))
+                    .containers(Arrays.asList(new V1Container()
+                        .name("create-weblogic-domain-onpv-container")
+                        .image(WLS_BASE_IMAGE_NAME + ":" + WLS_BASE_IMAGE_TAG)
+                        .imagePullPolicy("IfNotPresent")
+                        .ports(Arrays.asList(new V1ContainerPort()
+                            .containerPort(7001)))
+                        .volumeMounts(Arrays.asList(
+                            new V1VolumeMount()
+                                .name("create-weblogic-domain-job-cm-volume") // domain creation scripts volume
+                                .mountPath("/u01/weblogic"), // availble under /u01/weblogic inside pod
+                            new V1VolumeMount()
+                                .name(pvName) // location to write domain
+                                .mountPath("/shared"))) // mounted under /shared inside pod
+                        .addCommandItem("/bin/sh") //call wlst.sh script with py and properties file
+                        .addArgsItem("/u01/oracle/oracle_common/common/bin/wlst.sh")
+                        .addArgsItem("/u01/weblogic/create-domain.py")
+                        .addArgsItem("-skipWLSModuleScanning")
+                        .addArgsItem("-loadProperties")
+                        .addArgsItem("/u01/weblogic/domain.properties")))
+                    .volumes(Arrays.asList(
+                        new V1Volume()
+                            .name(pvName)
+                            .persistentVolumeClaim(
+                                new V1PersistentVolumeClaimVolumeSource()
+                                    .claimName(pvcName)),
+                        new V1Volume()
+                            .name("create-weblogic-domain-job-cm-volume")
+                            .configMap(
+                                new V1ConfigMapVolumeSource()
+                                    .name(domainScriptCM))))  //config map containing domain scripts
+                    .imagePullSecrets(Arrays.asList(
+                        new V1LocalObjectReference()
+                            .name(OCR_SECRET_NAME))))));
+    String jobName = assertDoesNotThrow(() -> TestActions
+        .createNamespacedJob(jobBody), "Domain creation job failed");
```

Comment:
you can static import TestActions.createNamespacedJob here

Reply:
fixed